### PR TITLE
Add filter for private and common runtimes

### PR DESCRIPTION
**Pull request by jingweno**
Minor improvements: Add flag `--private` to shows only private runtimes and flag `--common` to show only common runtimes.

/cc @heroku/dogwood @heroku/cedar

### CODE REVIEW

File: commands/regions/index.js [hunk 1]
```diff
@@ -5,9 +5,16 @@ const co = require('co')
 
 function * run (context, heroku) {
   const sortBy = require('lodash.sortby')
+  const filter = require('lodash.filter')
```
Comment (dickeyxxx) [inline commands/regions/index.js]: you don't need lodash for this. [].filter() is in es6.

Comment (jingweno) [inline commands/regions/index.js]: I'm aware of that. Being lazy :). I can fix it.

Comment (jingweno) [inline commands/regions/index.js]: Fixed in https://github.com/heroku/heroku-apps/pull/117/commits/6cd7ccb70ebb0fd594008b14e51be42a2089fc6f.

**Comment by amerine** [comment]
I like it!

File: commands/regions/index.js [hunk 2]
```diff
@@ -7,7 +7,13 @@ function * run (context, heroku) {
   const sortBy = require('lodash.sortby')
 
   let regions = yield heroku.get('/regions')
+  if (context.flags.private) {
+    regions = regions.filter((region) => { return region.private_capable })
```
Comment (dickeyxxx) [inline commands/regions/index.js]: minor nit, this could be written as:

```js
regions = regions.filter(region => region.private_capable)
```

**Comment by dickeyxxx** [comment]
code lgtm

File: commands/regions/index.js (hunk 2 above)
Comment (jingweno) [inline commands/regions/index.js]: TIL. :heart:: https://github.com/heroku/heroku-apps/pull/117/commits/f0114443a5a7cbbbb8f3bc9d6cb2ac52a8c55645

**Comment by jingweno** [comment]
The space creation experience I would like to have is:

```
$ h regions --private
ID         Location                 Runtime
─────────  ───────────────────────  ──────────────
frankfurt  Frankfurt, Germany       Private Spaces
oregon     Oregon, United States    Private Spaces
sydney     Sydney, Australia        Private Spaces
tokyo      Tokyo, Japan             Private Spaces
virginia   Virginia, United States  Private Spaces
$ h spaces:create --region tokoyo
...
```

Currently `h regions` show all and it's a bit noisy.

**Pull request merged**